monero-outreach-docs/translations/es/storyboard/story58.md

```diff
@@ -0,0 +1,67 @@
+con el que cuentan. No hay equipos de apoyo de pagos o servicio al cliente. Monero Monero tiene una comunidad en Telegram de 9k+ miembros que está activa 365 días y recibe cualquier pregunta, sin importar
+tu conocimiento técnico.
+
+Con un fuerte equipo de ingenieros, una grande comunidad y una historia implementando gran tecnología, Monero se mantiene aparte de monedas que dependen principalmente en tácticas de marketing.
+Nuevos proyectos son primero mostrados en el [foro de la comunidad](https://forum.getmonero.org/). Cualquiera puede comentar, votar y prometer sus propias monedas Monero para financiar sus proyectos favoritos. Una vez que un proyecto está en marcha, cualquiera puede ver los
+hitos y productos de trabajo. Esto mantiene al equipo del proyecto responsable y permite a los patrocinadores ver los resultados de los fondos del proyecto. Puedes ver exactamente cómo 
+tus contribuciones a Monero conducen a un resultado final.
+
+La organización de Monero yace en un fuerte contraste a monedas centralizadas con un liderazgo estricto. Los esfuerzos de monedas centralizadas a menudo rechazan las contribuciones de desarrolladores externos, incluso si el código es bueno.
+Los egos pueden salirse de control y el drama llegar a niveles catastróficos, con buenos desarrolladores saliendo e inversionistas perdiendo confianza. La estructura de Monero se mantiene fiel al espíritu descentralizado de forma que
+cualquiera es bienvenido a contribuir y su trabajo a menudo encuentra su camino en el código base, sitio web o documentación. Si hay algún problema con contribuciones, desarrolladores le hacen saber a los autores en una 
+manera gentil -- porque otra parte del espírito de Monero es "¡Sé excelente con los demás!"
+
+### Monero Research Lab - En camino hacia la mejor tecnología
+
+[El Laboratorio de Investigación de Monero](https://www.getmonero.org/es/resources/research-lab/) es un brazo de la comunidad en donde algunos de los investigadores de seguridad mas brillantes, matemáticos y desarrolladores crean soluciones de punta para privacidad y seguridad.
+
+
+
+
+## Bienvenidos Todos los Mineros
+
+Como el equipo de minería ASIC ha dominado el algoritmo de prueba-de-trabajo de Bitcoin, muchos mineros en-casa han sido dejados de lado. Esto deja la minería abierta sólo a unos pocos con mucho dinero y recursos para beneficiarse de la minería de Bitcoin. 
+
+Mientras que la minería ASIC se ha convertido en algo común para Bitcoin, Monero actualiza frecuentemente su algoritmo de prueba-de-trabajo para desalentar al equipo ASIC. Con este acercamiento, integrados CPU y GPU pueden permanecer competitivos y rentables. Esta causa ha sido y 
+seguirá siendo la promesa de Monero a la comunidad.
+
+La última mejora de núcleo de Monero incluye un algoritmo de punta Bulletproofs. La actualización mejora dramáticamente la privacidad, velocidad y costos de transacción de Monero. Todos estos nuevos beneficios harán que Monero destaque

```

Comment:
I wasn't very sure about the meaning of Bulletproofs here, I can see that it's just the name of the algorithm? And about the sentence, with just the _a medida que_ added it would be better, thanks.